### ISSUE TITLE
sh file

### ISSUE THREAD

**Issue by asavage8** [nigroup/IF_extension#1]
On the read me you said that you can compile the sh script on windows.  Do you have an example file of how to do this

**Comment by F-A** [comment]
No, you cannot compile the .sh script on windows (not that I know of at least).
The NEURON mod files need to be compiled in a given order and the .sh script is just a tool in order to compile them is te correct order.
Under windows, you should simply compile the mod by yourself in the same order as present in the script. Looking at the [script](https://github.com/nigroup/IF_extension/blob/master/compile_mod.sh), you should compile the mods in this order:
* Mod_files/fsin.mod
* Mod_files/xtra.mod
* Mod_files/IClampOU.mod
* Mod_files/Spikeout.mod
